pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_config_repo.adoc

```diff
@@ -1,15 +1,19 @@
 [id="configuring-a-repository-as-a-content-source-for-pantheon_{context}"]
 
-= Configuring a repository as a content source for {ProductShortName}
+= Configuring your documentation repository as a content source for {ProductShortName}
 
 [role="_abstract"]
-In order to enable a repository as a content source for {ProductShortName}, you need to configure the repository by adding a configuration file to the root directory.
+To enable a repository as a content source for {ProductShortName}, you must configure the repository by adding a configuration file to the root directory.
+
+.Prerequisites
+* A repository that contains your documentation.
+* You have a server URL from {ProductShortName} administrator.
```

Comment:
Nice addition. Why don't you use parallel language for these bullets: Either "You have" for both or neither?

Reply:
Yes, good point.